### ISSUE TITLE
McAfee warning

### ISSUE THREAD

**Issue by oaksprout** [valory-xyz/olas-operate-app#305]
https://discord.com/channels/899649805582737479/1244588374736502847/1278970717018521621

**Comment by truemiller** [comment]
Yeah, I'm getting spammed with Avast popups too on the Windows release, though they resolve to being safe after some time period.

@oaksprout 
We should look into:
- Windows code signing
- Automating AV submissions to major vendors -- Norton, Avast, McAfee, NOD32, VirusTotal -- once
release completes

**Comment by oaksprout** [comment]
Yikes, OK

**Comment by truemiller** [comment]
an update on the code signing: 
the bundled binaries should all be code-signed when using `electron-builder` for the release as we already do.

it's unlikely that we'll need to code-sign separately.
